test/acceptance/features/bindAppToService.feature

```diff
@@ -217,3 +217,56 @@ Feature: Bind an application to a service
         And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding Request "binding-request-empty-app" should be changed to "False"
         And jq ".status.conditions[] | select(.type=="InjectionReady").reason" of Service Binding Request "binding-request-empty-app" should be changed to "EmptyApplicationSelector"
 
+    Scenario: Bind backend service with application using bindable information specifed in CSV
+        Given Backend service "some-backend-service.v0.1.0" CSV is installed
+        * The CRD "backservs.service.example.com" is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backservs.service.example.com
+            spec:
+                group: service.example.com
+                versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                scope: Namespaced
+                names:
+                    plural: backservs
+                    singular: backserv
+                    kind: Backserv
+                    shortNames:
+                    - bs
+            """
+        * The application CR "csv-demo-backend" is present
+            """
+            apiVersion: service.example.com/v1
+            kind: Backserv
+            metadata:
+                name: csv-demo-backend
+            spec:
+                svcName: csv-demo-backend
+            """
+        * Imported Nodejs application "csv-demo-app" is running
+        When Service Binding Request is applied to connect the database and the application
+            """
+            apiVersion: apps.openshift.io/v1alpha1
+            kind: ServiceBindingRequest
+            metadata:
+                name: csv-demo-service-binding-request
+            spec:
+                applicationSelector:
+                    resourceRef: csv-demo-app
+                    group: apps
+                    version: v1
+                    resource: deployments
+                backingServiceSelector:
+                    group: service.example.com
+                    version: v1
+                    kind: Backserv
+                    resourceRef: csv-demo-backend
+            """
+        Then "csv-demo-app" deployment must contain secret ref "[map[secretRef:map[name:csv-demo-service-binding-request]]]"
```

Comment:
this check is not enough - we want to check if the secret contains values defined by binding declared in CSV. The current check does not tell us that at all.

Reply:
Removed this step.